mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -69,32 +79,32 @@ class ExperimentView extends Component {
 
   render() {
     const { experiment_id, name, artifact_location } = this.props.experiment;
-    const {
+    let {
```

Comment:
is it possible to keep only `paramKeyList` and `metricKeyList` as the `let` and the others still as constants?